examples/00-load/create-explicit-structured-grid.py

```diff
@@ -7,6 +7,8 @@
 Create an explicit structured grid from NumPy arrays using
 :class:`pyvista.ExplicitStructuredGrid`.
 
+.. tags:: Load
```

Comment:
I usually see lowercase tags everywhere, which includes the matplotlib gallery:
![image](https://github.com/user-attachments/assets/cdc481c2-17f4-4acd-8b3b-b80d0aa4d533)

My recommendation would be to lowercase all of our tags.

Reply:
All tags have been changed to lowercase and moved to the end of the file to match the Matplotlib format.

Reply:
Thanks a lot. Is it possible to trigger another docs deployment? If not, I can probably download the built artifact.

Reply:
@story645 I'm not sure what "cares about" means, but [the deployment preview](https://66928253b61ecdfab0f3c1f6--pyvista-dev.netlify.app/) still shows the initial state, and both the tag list and per-example tags respect the casing:
![Screenshot_20240718_094028_Opera](https://github.com/user-attachments/assets/94b64c7e-d4ca-469b-ba42-fa8a8e0b6f9e)
![Screenshot_20240718_094055_Opera](https://github.com/user-attachments/assets/cee75c56-da1c-469e-bdf3-ad947142ccaa)

And what are multiline tags?

Reply:
I see, thanks!